integration/tests/sample_data_test.go

```diff
@@ -65,10 +65,43 @@ func SampleAccountOwner() openapi.SetAccountOwner {
 		Email: "jsmith@atlassian.com",
 		Valid: true,
 	}
+	dale := openapi.SetPerson{
+		Name:  "Dale Cooper",
+		Login: "dcooper",
+		Email: "dcooper@atlassian.com",
+		Valid: true,
+	}
 	accountOwner := openapi.SetAccountOwner{
 		AccountId: accountID,
 		Owner:     alice,
-		Champions: []openapi.SetPerson{alice, john},
+		Champions: []openapi.SetPerson{john, dale, alice},
+	}
+	return accountOwner
+}
+
```

Comment:
Had to create another struct here for `AccountOwner` on GET requests (to validate response) since it is different from `SetAccountOwner`